app/services/schools/generate_chart_configuration.rb

```diff
@@ -0,0 +1,57 @@
+require 'dashboard'
+
+module Schools
+  class GenerateChartConfiguration
+    def initialize(
+        school,
+        aggregated_meter_collection,
```

Comment:
We could aggregate this here rather than passing it in?

Reply:
Prefer to pass one in as it's an expensive process. If you do it inside you can't then re-use it, or pass in a pre-calculated one.